src/webxr/OculusHandPointerModel.js

```diff
@@ -41,15 +42,30 @@ class OculusHandPointerModel extends THREE.Object3D {
 
     this.raycaster = null
 
-    hand.addEventListener('connected', (event) => {
-      const xrInputSource = event.data
-      if (xrInputSource.hand) {
-        this.visible = true
-        this.xrInputSource = xrInputSource
+    this._onConnected = this._onConnected.bind(this)
+    this._onDisconnected = this._onDisconnected.bind(this)
+    this.hand.addEventListener('connected', this._onConnected)
+    this.hand.addEventListener('disconnect', this._onDisconnected)
```

Comment:
it looks like the event type should be `disconnected`
```suggestion
    this.hand.addEventListener('disconnected', this._onDisconnected)
```